addon/components/nrg-modal.js

```diff
@@ -51,6 +51,41 @@ export default class NrgModal extends Component {
     return this.args.lightbox ?? false;
   }
 
+  get flyout() {
+    return this.args.flyout ?? false;
+  }
+
+  get stackable() {
+    return this.args.stackable ?? this.args.flyout ?? false;
+  }
+
+  get position() {
+    if (!this.flyout) {
+      return '';
+    }
+    return this.args.position ?? 'left';
+  }
+
+  get type() {
+    if (this.flyout) {
+      return 'flyout';
+    }
+    if (this.lightbox) {
+      return 'lightbox';
+    }
+    if (this.sidebar) {
+      return 'sidebar';
+    }
+    return 'modal';
+  }
```

Comment:
I didn't see this used anywhere, so I think it can probably be removed.

Reply:
I end up using this is my animation follow-up PR